src/main/java/domain/Player.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+public class Player extends User {
+
+    private static final int NAME_MAX_LENGTH = 5;
+    private static final int BLACK_JACK_SCORE = 21;
+    private static final String DEALER_NAME = "딜러";
+
+    public Player(final String name) {
+        validateName(name);
+        this.name = name;
+    }
```

Comment:
```suggestion
    public Player(final String name) {
        super(name);
    }
```
검증로직은 User에서 처리해도 되겠네요

Reply:
저도 처음엔 User에서 검증을 하고싶었으나 Player에서 했던 이유는 들어온 이름이 혹여 "딜러"가 될 수 있기 때문이었습니다.
스스로와의 합의를 통해 "딜러"이름 검증은 하지 않는 것으로 결정했습니다. 혹시 User에서 들어오는 모든 이름을 검증하는 상황에서 Player의 이름만 검증할 수 있는 방법이 있을까요>?